General/ExportFormats.md

```diff
@@ -87,13 +87,13 @@ Metacello new
 
 ## FileTree metadata less 
 
-FileTree ([https://github.com/dalehenrich/filetree](https://github.com/dalehenrich/filetree)) is the first export format planned to use Pharo with Git that was integrated. It had the first version with a lot of metadata (see section Filetree metadata full) then a new version with metadata less format. 
+FileTree ([https://github.com/dalehenrich/filetree](https://github.com/dalehenrich/filetree)) is the first export format that was integrated in the goal to use Pharo with Git. The first version had a lot of metadata (see section Filetree metadata full), the second was a new version with metadata less format. 
 
-This format exports one file per methods. This can cause trouble with Windows because Windows has a maximum path and file name length of 255 characters. This limit can easily be reached with FileTree. 
+This format exports one file per method. This can cause trouble with Windows because Windows has a maximum path and file name length of 255 characters. This limit can easily be reached with FileTree. 
 
 Another problem of the FileTree format is that most Pharo's methods are short and creating one file by method waste a lot of space because the physical representation of files on a drive has a minimal size (4 KiB on NTFS file systems) that is reached by most of Pharo's method. 
 
-Its advantage compared to FileTree metadata full is that there is no problem during merge because of metadata.
+Its advantage compared to FileTree metadata full is that there is no problem during merge of the source code thanks to the metadata.
```

Comment:
@VincentBlondeau No thanks to metadata since there is none. And that's what make it easier ;)